Automattic/newspack-block-theme
# feat: update fonts from system to inter and from monospace to jetbrainsmono

### All Submissions:






### Changes proposed in this Pull Request:



We're ditching system font (but keep it as a backup) for [Inter](https://rsms.me/inter/). Same with the generic monospace font, we're switching to [JetBrains Mono](https://www.jetbrains.com/lp/mono/).

![e g](https://github.com/Automattic/newspack-block-theme/assets/177929/696c7822-4e45-4abc-9d44-bae5d54d726f)

### How to test the changes in this Pull Request:

1. Make sure you don't have Inter or JetBrains Monospace installed on your machine.
2. Open the site and edit a page. You should see system font.
3. Add a code block. You should see monospace font.
4. Switch to this branch.
5. You should see Inter and JetBrainsMono.

### Other information:
